pika/connection.py

```diff
@@ -1545,7 +1594,7 @@ def _on_terminate(self, reason_code, reason_text):
         self._remove_heartbeat()
 
         # Remove connection management callbacks
-        # TODO: This call was moved here verbatim from legacy code and the
+        # TODO This call was moved here verbatim from legacy code and the
         # following doesn't seem to be right: `Connection.Open` here is
         # unexpected, we don't appear to ever register it, and the broker
         # shouldn't be sending `Connection.Open` to us, anyway.
```

Comment:
IIRC the original implementation was fairly agnostic to which side of the conversation it was on.